integration-cli/docker_cli_run_unix_test.go

```diff
@@ -265,6 +265,13 @@ func (s *DockerSuite) TestRunWithBlkioInvalidWeight(c *check.C) {
 	}
 }
 
+func (s *DockerSuite) TestRunWithBlkioInvalidWeightDevice(c *check.C) {
+	cmd := exec.Command(dockerBinary, "run", "--blkio-weight-device", "/dev/sda:5", "busybox", "true")
+	if _, err := runCommand(cmd); err == nil {
```

Comment:
+1 to dockercmd and checking weights... and maybe some test with dd or something.